src/citrine/resources/table_config.py

```diff
@@ -404,21 +405,35 @@ def get_job_status(self, job_id: str):
         response: dict = self.session.get_resource(path=path)
         return JobStatusResponse.build(response)
 
-    def preview(self, table_config: TableConfig, preview_roots: List[LinkByUID]) -> dict:
+    def preview(self, *,
+                table_config: TableConfig,
+                preview_materials: List[LinkByUID] = None,
+                preview_roots: List[LinkByUID] = None
+                ) -> dict:
         """[ALPHA] Preview a Table Config on an explicit set of terminal materials.
 
         Parameters
         ----------
         table_config: TableConfig
             Table Config to preview
-        preview_roots: List[LinkByUID]
+        preview_materials: List[LinkByUID]
             List of links to the material runs to use as terminal materials in the preview
+        preview_roots: List[LinkByUID]
+            [DEPRECATED] Use preview_materials instead
 
         """
+        if preview_materials is None and preview_roots is None:
+            raise ValueError("Must specify preview materials")
+        if preview_materials is not None and preview_roots is not None:
+            raise ValueError("Cannot specify both preview_materials and preview_roots")
+        if preview_roots is not None:
+            preview_materials = preview_roots
+            warn("preview_roots argument is deprecated in favor of preview_materials",
+                 DeprecationWarning)
```

Comment:
```suggestion
        if preview_roots is not None:
            warn("preview_roots argument is deprecated in favor of preview_materials",
                 DeprecationWarning)
            if preview_materials is None:
                preview_materials = preview_roots
            else:
                raise ValueError("Cannot specify both preview_materials and preview_roots")
        elif preview_materials is None:
                raise ValueError("Must specify preview materials")
```
I like that the above warns any time the user uses the `preview_roots` argument and, when it is removed, we delete a coherent block of text.